### DISCUSSION TITLE
va-tree-view Component: Is it possible to highlight the background color for selected node in the tree.

### DISCUSSION THREAD

**Discussion by kumarimanjari** [epicmaxco/vuestic-ui#3448] [Q&A]
Hi Team,

I have a question around highlighting (adding the background-color property) for the selected node in the Tree component.
Is this possible?

Do let me know your thoughts around it.

Thanks & Regards,
Manjari Kumari

**Comment by kumarimanjari** [comment]
Hello @m0ksem ,

Do you have any solution for this?

Thanks & Regards,
Manjari Kumari

**Reply by m0ksem** [reply]
Hi. I think there is no way yet. I'll make a feature request for it. If you want, you can implement it. Should be easy.

**Reply by kumarimanjari** [reply]
@m0ksem - Could you please add it as a feature request and also it would be really nice if you could let me know how to implement it as well

**Reply by m0ksem** [reply]
Here is the issue https://github.com/epicmaxco/vuestic-ui/issues/3456
